scripts/l1_node.sh

```diff
@@ -56,6 +56,12 @@ allorad \
     --minimum-gas-prices=0${DENOM} \
     --rpc.laddr=tcp://0.0.0.0:26657 \
     --p2p.seeds=$SEEDS \
-    --log_level "*:error,state:info,server:info,rewards:debug,inference_synthesis:debug,topic_handler:debug" 
-
-
+    --log_level "*:error,state:info,server:info,rewards:debug,inference_synthesis:debug,topic_handler:debug" \
```

Comment:
There are duplicates here. need cleanup and just pass one flag each. FYI this is mostly what nops use to standup external val